.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -6,7 +6,12 @@ labels: type/enhancement, area/documentation
 assignees: ''
 
 ---
+### Tell us about the documentation you'd like us to add or update.
+It can be as pedantic as a typo or as titanic as an entire tutorial request.
 
-## Tell us about the documentation you'd like us to add or update
 
-## Are there any related areas of documentation that currently exist? Could you point to them? 
+### If applicable, add links to the relevant docs that should be updated
+- [Link to relevant docs page](https://docs.airbyte.io)
+
+### Finally, add this issue to the `Documentation Roadmap` GitHub project on the right ->
+This helps us prioritize your issue and will make sure it's done quickly!
```

Comment:
I think contributors can modify this themselves.
We can improve this script https://github.com/airbytehq/airbyte/blob/master/.github/workflows/project.yaml

Reply:
Oh we do have the github action auto assign, ok I'll throw it in there.

Reply:
Ok @marcosmarxm done!